### ISSUE TITLE
Discord gives "Invalid application interaction command" error when I run a slash command

### ISSUE THREAD

**Issue by open-antux** [AlexzanderFlores/WOKCommands#126]
I have several slash commands, but often when I run my discord bot and run one of the commands, discord gives me an Invalid application interaction command and the only thing that I can do is stop the bot, wait a while and rerun the bot.
This is an example of command:
```javascript
const { MessageEmbed } = require('discord.js');

module.exports = {
    name: 'info',
    alias: ['i'],
    category: 'Info',
    description: 'Show info.',
    slash: true,
    
    callback: async ({}) => {
        return new MessageEmbed()
            .setTitle('x')
            .setAuthor('x')
            .setColor('0ffd0')
            .setDescription('x')
    }
}
```
The rest of the commands are very similar to this one. How can I solve this?
If could help, this is my index.js
```javascript
const Discord = require('discord.js');
const WOKCommands = require('wokcommands');

const Config = require('./config.json');

const client = new Discord.Client({ partials: ['MESSAGE', 'REACTION'] });

client.on('ready', () => {
    new WOKCommands(client, {
        commandsDir: 'commands',
        showWarns: true,
        del: -1,
        ignoreBots: true,
    }).setDefaultPrefix(Config.prefix);

    console.log('Bot is ready!');
})

client.login(Config.token);
```

**Comment by Wolletje01** [comment]
Can you send the error.

**Comment by daniieljc** [comment]
The same thing happens to me, the first time you run the discord command it sends you that error but from there it works correctly

**Comment by open-antux** [comment]
I also tried to create and use slash commands by me without using the WOKcommands handler, and I noticed that the error persist but it start to work again when the bot goes offline and return online.

@Wolletje01 my bot don't give any error, is my discord that give me `Invalid application interaction command` error.

That's an offtopic question: Where can I find some cool docs about interactions?

Thanks to everyone
